client.go

```diff
@@ -56,6 +56,8 @@ type Client interface {
 	// partition. Offline replicas are replicas which are offline
 	OfflineReplicas(topic string, partitionID int32) ([]int32, error)
 
+	RefreshBrokersWithNewSeedBrokers(seedBrokers []*Broker) error
```

Comment:
all the other functions have descriptions so maybe we should add one here too

Reply:
maybe the function could be called `RefreshBrokers` and the rest could go in the description?